service/server.go

```diff
@@ -43,6 +43,7 @@ func serve() {
 
 	//Metadata
 	router.HandleFunc("/v1/meta/{name}", res.MetaGetHandler).Methods("GET")
+	router.HandleFunc("/v1/metacreate", res.MetaCreateHandler).Methods("POST")
```

Comment:
i think it would be more RESTful if you keep the name of the route as `/v1/meta` and simply use the fact that it's a POST request to indication intention to create a new item

Reply:
good idea